network/reset_wifi.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sh /usr/sbin/firstboot-connman-setup
+/usr/bin/connmanctl tether wifi off
+sleep 1
```

Comment:
Which also means the approach we are taking here is not the best one since 1 sec sleeping is an arbitrary value.  It's ok for now though.  In future we should probably do a single client script in python.